src/components/EditServerDetails.js

```diff
@@ -26,7 +26,7 @@ class EditServerDetails extends Component {
   constructor(props) {
     super(props);
 
-    this.IS_INSTALL = this.props.mode ? false : true;
+    this.IS_INSTALL = this.props.mode === undefined ? false : true;
```

Comment:
I think this logic is the reverse of what you're intending, will pull it down and try it out